nevergrad/benchmark/experiments.py

```diff
@@ -1073,6 +1074,47 @@ def rocket(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                             yield xp
 
 
+@registry.register
+def gym_multi(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Gym simulator. Maximize reward.
+    Many distinct problems."""
+    if os.name == "nt":
+        raise ng.errors.UnsupportedExperiment("Windows is not supported")
```

Comment:
this should go inside the ExperimentFunction directly, this way the check happens only once, in the experiment function, not in experiment.py, nor in the tests

Reply:
Done.